examples/04_vcb_tool/ex_vcb_tool.rb

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Trimble Inc
+# Licensed under the MIT license
+
+# This demonstrate how to improve a custom Ruby tool to allow precise text input
+# in the measurement bar (aka VCB).
+
+require 'sketchup.rb'
+require 'extensions.rb'
+
+module Examples
+  module VCBTool
+
+    unless file_loaded?(__FILE__)
+      ex = SketchupExtension.new('Measurement Bar', 'ex_vcb_tool/main')
+      ex.description = 'SketchUp Ruby API example of tool Measuremt Bar support.'
```

Comment:
"Measuremt" => "Measurement"

Reply:
Side note: Maybe worth updating our API docs to explain that in the UI the VCB is called "Measurement Bar"...